<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCaisse</class>
 <widget class="QDialog" name="DialogCaisse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>1€</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>0.50€</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>0.20€</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>500€</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>0.10€</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>0.05€</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>0.02€</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Total Théorique</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Total Réel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>2€</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>0.01€</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_5"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_6"/>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBox_7"/>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBox_8"/>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="spinBox_9"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>200€</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>100€</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>50€</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>20€</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>10€</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>5€</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSpinBox" name="spinBox_10"/>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBox_11"/>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinBox_12"/>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="spinBox_13"/>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="spinBox_14"/>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="spinBox_15"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCaisse</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCaisse</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
